src/app/layout.tsx

```diff
@@ -16,8 +18,12 @@ export default function RootLayout({
   children: React.ReactNode;
 }) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
-    </html>
+    <SkeletonTheme>
+      <html lang="en">
+        <body className={inter.className}>
+          <CheckRefreshProvider>{children}</CheckRefreshProvider>
+        </body>
+      </html>
+    </SkeletonTheme>
```

Comment:
El `SkeletonTheme` sería mejor ponerlo a nivel del `body` así cómo está el provider
```suggestion
    
      <html lang="en">
        <body className={inter.className}>
        <SkeletonTheme>
          <CheckRefreshProvider>{children}</CheckRefreshProvider>
           </SkeletonTheme>
        </body>
      </html>
   
```

Reply:
Perfecto, ya lo cambie